### PR TITLE
Make login form look more like on android

### PR DESCRIPTION
**Pull request by Jikstra**
What's missing:
- [x] If a field is selected or filled out, the placeholder/label is not shown. Currently this makes the ux a bit worse, will implement it later.

After:
Screenshot_20190605_001548
Screenshot_20190605_001711

Before:
deltachat-old-login-1
deltachat-old-login-2

### CODE REVIEW

**Comment by ralphtheninja** [comment]
LGTM

**Comment by Jikstra** [comment]
@ralphtheninja this is ready for a final review. This pr also introduces the ability to use the import/export syntax which is handy for the frontend.

**Comment by Simon-Laux** [comment]
Can't test the functionality right now, because the compatibility from newest dc-node to desktop was broken https://github.com/deltachat/deltachat-node/issues/341
The optics look better though.:+1:

**Comment by Simon-Laux** [comment]
> This pr also introduces the ability to use the import/export syntax which is handy for the frontend.

:heart:

File: src/renderer/components/Login-Styles.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+          onFocus={onFocus}
+          onBlur={onBlur}
+          rightElement={props.rightElement}
+        />
+      </FormGroup>
+    </DeltaInputWrapper>
+  )
+})
+
+export const DeltaPasswordInput = React.memo((props) => {
```
Comment (ralphtheninja) [inline src/renderer/components/Login-Styles.js]: What's this `.memo()` thingie?

Comment (Jikstra) [reply src/renderer/components/Login-Styles.js]: It's memoizing, so this component doesn't get rerendered/reexecuted if the props stay the same.

Comment (ralphtheninja) [reply src/renderer/components/Login-Styles.js]: Cool!

File: src/renderer/components/Login.js [hunk 2]
```diff
@@ -64,11 +65,17 @@ class Login extends React.Component {
...
 
   handleCredentialsChange (event) {
-    let stateCredentials = Object.assign(this.state.credentials, { [event.target.id]: event.target.value })
-    this.setState(stateCredentials)
+    let updatedState = update(this.state, {
+      credentials: {
+        [event.target.id]: { $set: event.target.value }
+      }
+    })
+    console.log('handleCredentialsChange', updatedState)
```
Comment (ralphtheninja) [inline src/renderer/components/Login.js]: `console.log`

File: src/renderer/components/Login.js [hunk 3]
```diff
@@ -64,11 +65,17 @@ class Login extends React.Component {
...
+    this.setState(updatedState)
   }
 
   handleSubmit (event) {
+    console.log(event)
```
Comment (ralphtheninja) [inline src/renderer/components/Login.js]: And here

**Pull request merged**